plugins/hls-explicit-record-fields-plugin/src/Ide/Plugin/ExplicitFields.hs

```diff
@@ -249,8 +321,8 @@ instance Show CollectNamesResult where
 type instance RuleResult CollectNames = CollectNamesResult
 
 data RecordInfo
-  = RecordInfoPat RealSrcSpan (Pat (GhcPass 'Renamed))
-  | RecordInfoCon RealSrcSpan (HsExpr (GhcPass 'Renamed))
+  = RecordInfoPat RealSrcSpan (Pat GhcTc)
+  | RecordInfoCon RealSrcSpan (HsExpr GhcTc)
```

Comment:
Ah, does that mean this plugin now only works if the source file can be typechecked? And previously, it already worked if the source was renamed?

Reply:
Yes